Resources/public/mapbender.element.coordinatesutility.js

```diff
@@ -502,12 +502,17 @@
                 this.highlightLayer.olLayer.addFeatures(this.feature);
 
                 var lonLat = new OpenLayers.LonLat(this.lon, this.lat);
+                var zoomlevel = this.options.zoomlevel;
+                var olMapMaxZoom = this.mbMap.map.olMap.numZoomLevels;
 
-                this.mbMap.map.olMap.setCenter(lonLat, this.ZOOM);
+                if (zoomlevel && zoomlevel < olMapMaxZoom) {
+                    this.mbMap.map.olMap.setCenter(lonLat, zoomlevel);
+                } else {
+                    this.mbMap.map.olMap.setCenter(lonLat, this.ZOOM);
+                }
```

Comment:
If numZoomLevels <= both the hardcoded this.ZOOM and the configured this.options.zoomlevel, this first prevents using an inacceptable zoom level, then sets one anyway.   
This should be `Math.min(numZoomLevels - 1, this.options.zoomlevel, this.ZOOM)` if we're going to keep the change localized.

In actuality, `this.ZOOM` is getting replaced with the configuration option and should go away. To remove it safely, one would have to provide a default value server-side in (hopefully) getPublicConfiguration for CoordinatesUtility Elements that have been configured and saved before the introduction of the change.